examples/Pyspark Kernel.ipynb

```diff
@@ -0,0 +1,686 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Features available with the new kernels\n",
+    "\n",
+    "By default Jupyter notebook comes with a `Python2` kernel. A kernel is a program that runs and interprets your code. sparkmagic provides two additional kernels that you can use with Jupyter. These are:\n",
+    "\n",
+    "1. **PySpark** (for applications written in Python). PySpark kernel exposes the Spark programming model to Python\n",
+    "2. **Spark** (for applications written in Scala)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "------------\n",
+    "## How do I use the new kernels? \n",
+    "\n",
+    "1. Create a notebook with the new kernels. Click **New**, and then click **PySpark** or **Spark**. \n",
```

Comment:
I think it's someone who wants to use Spark with Jupyter. Level of expertise greatly varies. Pros can skip these paragraphs while non-pros can read in depth.